qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_model.py

```diff
@@ -783,18 +814,36 @@ def _evaluate_plot_guess(self, plot_guess: bool) -> str:
 
     def _evaluate_function(self, fit_function: IFunction, output_workspace: str) -> str:
         """Evaluate the plot guess fit function and returns the name of the resulting guess workspace."""
+        if self.fitting_context.plot_guess_type == 'x from plot range':
+            start_x = self.current_start_x
+            end_x = self.current_end_x
+            points = 250
```

Comment:
why 250

Reply:
This is a holdover for now, I am in the middle of checking what the most effective way of using the parameters from the workspace

Reply:
fair enough. I would assume this would mean the same number of data points as my data, just evenly spaced

Reply:
I was assuming it was supposed to be the same data range and number of points as the origional data

Reply:
yep, just evenly spaced across that range